MAUI/DataGrid/export-to-pdf.md

```diff
@@ -1144,4 +1144,112 @@ private void PdfExport_CellExporting(object sender, DataGridCellPdfExportingEven
 {% endhighlight %}
 {% endtabs %}
 
-<img alt="Export DataGrid to PDF format with customized cell style" src="Images\export-to-pdf\maui-datagrid-style-based-on-column-name.png" width="689"/>
\ No newline at end of file
+<img alt="Export DataGrid to PDF format with customized cell style" src="Images\export-to-pdf\maui-datagrid-style-based-on-column-name.png" width="689"/>
+
+### Exporting DetailsView
+
+By default, [DetailsViewDataGrid](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DetailsViewDataGrid.html) will not be exported to PDF. You can export `DetailsViewDataGrid` by setting `CanExportDetailsView` property as true.
+
+```csharp
+MemoryStream stream = new MemoryStream();
+DataGridPdfExportingController pdfExport = new DataGridPdfExportingController();
+DataGridPdfExportingOption option = new DataGridPdfExportingOption();
+option.CanExportDetailsView = true;
+var pdfDoc = pdfExport.ExportToPdf(this.dataGrid, option);
+pdfDoc.Save(stream);
+pdfDoc.Close(true);
+SaveService saveService = new();
+saveService.SaveAndView("ExportFeature.pdf", "application/pdf", stream);
+```
+<img src="Images/export-to-pdf/maui-datagrid-exportdetailsview.png" alt="Maui DataGrid displays Nested dataGrid exported to PDF" width="404"/>
+
+By default, only expanded DetailsViewDataGrids only will be exported to PDF document. If you want to export all the DetailsViewDataGrids, you need to set `CanExportAllDetails` as true.
+
+```csharp
+MemoryStream stream = new MemoryStream();
+DataGridPdfExportingController pdfExport = new DataGridPdfExportingController();
+DataGridPdfExportingOption option = new DataGridPdfExportingOption();
+option.CanExportDetailsView = true;
+option.CanExportAllDetails = true;
+var pdfDoc = pdfExport.ExportToPdf(this.dataGrid, option);
+pdfDoc.Save(stream);
+pdfDoc.Close(true);
+SaveService saveService = new();
+saveService.SaveAndView("ExportFeature.pdf", "application/pdf", stream);
+```
+<img src="Images\export-to-pdf\maui-datagrid-exportalldetails.png" alt="Nested dataGrid exported to PDF" width="404"/>
+
+Here, first record only expanded in SfDataGrid. But all the DetailsViewDataGrid’s are shown in exported PDF document.
+
+You can customize its exporting operation by using `DataGridChildPdfExportingEventArgs`.
+
+By default, [DetailsViewDataGrid](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DetailsViewDataGrid.html) will be exported to PDF. You can customize its exporting operation by using `DataGridChildPdfExportingEventArgs`.
+
+## Excluding DetailsViewDataGrid while exporting
+You can exclude particular DetailsViewDataGrid while exporting, by using the DataGridChildExcelExportingEventArgs and `DataGridChildPdfExportingEventArgs.Cancel`.
```

Comment:
You can exclude particular DetailsViewDataGrid while exporting, by using the `DataGridChildPdfExportingEventArgs.Cancel`.